waterlog/src/Components/Map.js

```diff
@@ -150,17 +108,23 @@ class MapComponent extends Component {
           </div>
           <div className={`map-icon-div-layer2 ${this.state.simpleView ? "" : "invisible"}`}>
             <img
-              className="icon-home"
+              className={`icon-home + ${this.state.moreOptions ? "" : "invisible"}`}
               src={require("../images/heatmap_icon_blue.png")}
               alt="heat Toggle"
               onClick={() => { this.setState({ heatView: !this.state.heatView }) }} />
             <img
-              className="icon-home"
+              className={`icon-home + ${this.state.moreOptions ? "" : "invisible"}`}
               src={require("../images/recentre_icon_blue.png")}
               alt="re-center Map"
               onClick={() => {
-                const map = this.refs.map.leafletElement;
-                map.setView(mapOptions.centerPosition, mapOptions.defaultZoom)
+                this.refs.map.reCenter()
+              }} />
+            <img
+              className="icon-home"
+              src={require("../images/more_map_icon.png")}
```

Comment:
Please extract the images to a variable

Reply:
It works something like 
`import moreMapIcon from '../images/more_map_icon.png'`
then you can say 
src={moreMapIcon)

Reply:
imported images into variable